utils/deprecation.py

```diff
@@ -0,0 +1,9 @@
+
+
+from debtcollector.removals import remove
+
+
+removed_selenium = remove(
```

Comment:
I was meaning to have a base which was ```removal_version="framework 3.0"```, but then you would have the ```removed_selenium``` defined in the pytest selenium module, and allow people to decorate and supply a message in the decoration.